test/lib/ufe/testComboCmd.py

```diff
@@ -613,12 +613,8 @@ def testFallbackCases(self):
                 "xformOp:rotateZ", "!invert!xformOp:translate:pivot",
                 "xformOp:rotateXYZ:maya_fallback")))
 
-    # Name test such that it runs last.  Otherwise, it runs before 
-    # testRotateScalePivotCompensation(), and causes it to fail.  To be 
-    # investigated --- MAYA-108067.  Fails on Linux, both Python 2 and 3
-    # --- MAYA-108612.
     @unittest.skipIf(mayaUtils.previewReleaseVersion() < 121 or platform.system() == 'Linux', 'Fallback transform op handling only available in Maya Preview Release 121 or later.')
```

Comment:
Need to adjust the skipIf to remove the Linux condition.

Also, I wonder if you should just change the previewReleaseVersion() check to be <= 121, since your Maya-side fix will only be in PR122.